src/lib/ns/user/user.c

```diff
@@ -39,11 +39,32 @@
 #include "lib/config_parser.h"
 #include "lib/privilege.h"
 
-static int enabled = -1;
-
 int singularity_ns_user_enabled(void) {
-    singularity_message(DEBUG, "Checking user namespace enabled: %d\n", enabled);
-    return(enabled);
+    singularity_message(DEBUG, "Checking if user namespace is enabled.\n");
```

Comment:
Maybe a solution would be to have two user namespace functions `singularity_ns_user_enabled()` and `singularity_ns_user_allowed()`? Where one checks if you're *allowed* to run with namespaces, the other checks if you are *currently* running in a namespace?